src/app/layout.tsx

```diff
@@ -4,6 +4,8 @@ import { type Metadata } from "next";
 
 import { TRPCReactProvider } from "~/trpc/react";
 
+import AuthProvider from "./_components/NextAuth/AuthProvider";
```

Comment:
Always use path aliases over relative aliases, using the `~`. As the project scales, these will become spaghetti imports which are extremely difficult and painful to manage.

You can find the path it maps to in `tsconfig.json`. We're planning on changing that character to `@` later.

Using path aliases has lots of benefits, just to name a few:
- You as a developer don't have to relative pathing math to get to your component
- The file can be moved to anywhere else in the project, and the components won't break
    - Mass migrations and refactors are made possible through codemods. I will be demoing one in action when we change the character for the path alias.